stories/side-navigation/side-navigation.stories.js

```diff
@@ -1254,3 +1254,85 @@ nestedListWithoutLinks.parameters = {
         disable: true
     }
 };
+
+export const nestedListPopover = () => `<div class="fd-popover fd-popover--right" style="margin-bottom: 200px; margin-right: 200px; margin-left: 200px;">
+    <div class="fd-popover__control">
+        <ul class="fd-nested-list fd-nested-list--compact">
+            <li class="fd-nested-list__item">
+                <a class="fd-nested-list__link" href="#/">
+                    <i role="presentation" class="fd-nested-list__icon sap-icon--home"></i>
+                </a>
+            </li>
+        </ul>
+    </div>
+    <div class="fd-popover__popper" aria-hidden="false" id="popoverA11">
+        <ul class="fd-nested-list fd-nested-list--popover fd-nested-list--compact">
+            <li class="fd-nested-list__group-header fd-nested-list__group-header--popover">
+                Group Header 1
+            </li>
+            <li class="fd-nested-list__item fd-nested-list__item--popover">
```

Comment:
Is it necessary to have a modifier class on each element? Can't you use specificity? You have the `--popover` modifier on the parent element, you can then have something like:

```
.fd-nested-list--popover {
     .fd-nested-list__group-header { code overwrite}
     .fd-nested-list__item { code overwrite}
}
```

Reply:
using specificity now. Thanks for the `valuable` suggestion.